docs/docs/tutorial/contentful/index.mdx

```diff
@@ -0,0 +1,313 @@
+- serve: The serve script allows you to preview your build locally.
+- clean: The clean script removes the `public` and `.cache` directories.
+
+### 5. Contentful Space API Keys
+
+In the Contentful UI navigate to Settings > API Keys. Here you will find the Space ID and Content Delivery API access token.
+
+If you don't already have a Space created, you can [reference the Contentful docs](https://www.contentful.com/help/contentful-101/#step-2-create-a-space).
+
+### 6. gatsby-config.js
+
+Create a new file at the root of your project called `gatsby-config.js` and add the plugin config. Add your own Space ID and Content Delivery API access token from the previous step.
+
+```javascript
+// gatsby-config.js
+
+module.exports = {
+  plugins: [
+    ...{
+      resolve: "gatsby-source-contentful",
+      options: {
+        spaceId: "...",
+        accessToken: "...",
+      },
+    },
+    "gatsby-plugin-image",
+  ],
+}
+```
+
+_For production sites you should add these as environment variables. There's more information in the docs about how to do this: [Environment Variables](https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/)._
+
+### 7. Create Content Type
+
+In the Contentful UI navigate to Content Model and click add type. To start with, create one called Page.
+
+### 8. Add Fields to Content Type
+
+With a Content type created, you can now add fields.
+
+Add the following fields to the Page.
+
+- slug: The slug field is required and is of type Text, and Short text.
+- title: This title field is required and is of type Text, and Short text.
+- description: This description field is required and is of type Text, and Long text.
+
+Fields names are case sensitive. Double check all field names are lowercase.
+
+You can read more about Contentful's [Field types in the Contentful doc](https://www.contentful.com/help/contentful-glossary/#fields).
+
+### 9. Add Content to Fields
+
+In the Contentful UI navigate to Content to create a new page Page. Add content for the slug, title and description. When you're ready, publish your changes.
+
+Add the following content to create an Index or Home page.
+
+- slug: Set this to / for the index page of a site.
+- title: Add a title of the page.
+- description: Add a short description for the page.
+
+Repeat the same steps to create a Not Found page.
+
+- slug: Set this to /404 for the not found page of a site.
+- title: Add a title of the page.
+- description: Add a short description for the page.
+
+### 10. GraphiQL Explorer
+
+Run the following in your terminal to start the Gatsby development server.
+
+```shell
+npm run develop
+```
+
+With the development server running navigate to the `/__graphql` in your browser.
+[http://localhost:8000/\_\_graphql](http://localhost:8000/__graphql)
+
+This is a graphical representation of Gatsby's Data Layer.
+
+Notice the two data types:
+
+1. allContentfulPage
+2. contentfulPage
+
+Page is the name of the Content Type you created earlier..
```

Comment:
Not sure if i follow. If folks can't watch a video, would they also perhaps struggle with using GraphiQL, VS Code and the terminal?

Reply:
No, because those applications can be used with a screen reader. Things will get read out. The "Notice the two data types" for example isn't helpful for visually impaired people because it doesn't say where on the page this is, what they need to look out for. Screen readers read out everything, so giving context where something is, is necessary.

Reply:
The step before this instructs people to start the development server and open GraphiQL in the browser. If a user were to navigate through all the GraphiQL nodes they'd be able to notice both that have been mentioned in the video and text.

Reply:
The current tutorial shows how I imagine it to be done: https://www.gatsbyjs.com/docs/tutorial/part-4/#use-graphiql-to-explore-the-data-layer-and-write-graphql-queries

It establishes both in the alt text of the image but also in the "A closer look at the GraphiQL interface" section how the UI is built.

Then later in the text one can refer to the UI elements defined earlier:

> In the Explorer pane, open the dropdown for the site field.